site/plugin/plugin-how-it-works.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: How the Weave Net Docker Network Plugin Works
+title: How the Weave Net Docker Network Plugin (Legacy) Works
```

Comment:
Can we extend this doc to explain the three ways it can work now?  (i.e. two legacy and one v2)